spec/file_reference_spec.rb

```diff
@@ -54,8 +54,19 @@
 
     describe '#source_contents' do
       it 'gets the contents from the secrets repo' do
-        allow(File).to receive(:read).with(subject.secrets_repository_file_path).and_return('source contents')
-        expect(subject.source_contents).to eq('source contents')
+        set_circle_env(false) do
+          allow(File).to receive(:read).with(subject.secrets_repository_file_path).and_return('source contents')
+          expect(subject.source_contents).to eq('source contents')
+        end
+      end
+    end
+
+    describe '#source_contents_on_ci' do
```

Comment:
I'm sure what's the naming convention for RSpec, but since there's no `source_contents_on_ci` method, it might be misleading to use that name for this `describe` block. What do you think of:

```suggestion
    describe '#source_contents on ci' do
```

Reply:
👍